quantum/q1.py

```diff
@@ -0,0 +1,28 @@
+import qiskit as q
+
+"""
+Build a simple bare-minimum quantum
+circuit that starts with a single qubit
+(by default in state 0), runs the experiment
+1000 times, and finally prints the total 
+count of the states finally observed.
```

Comment:
```suggestion
Build a simple bare-minimum quantum circuit that starts with a single qubit (by
default, in state 0), runs the experiment 1000 times, and finally prints the total
count of the states finally observed.
```
Lines should be wrapped at 88 characters max.

Reply:
Pushed requested changes in a new commit. :+1: